packages/grid/x-data-grid-pro/src/DataGridPro/useDataGridProComponent.tsx

```diff
@@ -120,9 +120,9 @@ export const useDataGridProComponent = (
 
   useGridRowGrouping(apiRef, props);
   useGridTreeData(apiRef);
+  useGridDetailPanel(apiRef, props);
```

Comment:
Had to move it before `useGridKeyboardNavigation` to prevent `Space` moving focus to the bottom of the page

Reply:
Reverted